src/chrome/content/rules/Deredactie.be.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Deredactie.be">
+    <target host="deredactie.be" />
+    <target host="m.deredactie.be" />
```

Comment:
Please also add `hetvooruitzicht.deredactie.be` and a comment for non-working `www.deredactie.be`